smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HttpProtocolServiceClient.kt

```diff
@@ -74,27 +74,4 @@ open class HttpProtocolServiceClient(
             }
         }
     }
-
-    private fun renderConfig(serviceSymbol: Symbol) {
```

Comment:
This configuration is no longer code-generated and a runtime type is used to configure clients instead.